CommuniTEAM/CommuniTEA
# [TEA-20] Migrations: Users Tables

### In this PR, I'm adding/changing the following:

- Added two (2) new migration files: `create_users_tables` and `populate_user_roles`
    - Three (3) new tables were created: `users`, `user_roles`, and `user_favorite_teas`
    - The `user_roles` table was pre-populated with the three roles (user, business, and admin)
- Removed earlier example code (sql files and api endpoints) related to users to make way for API development

### Please do the following to test my code:

1. Stop and delete docker containers and **delete the `communitea-db` volume**
2. Create a new `communitea-db` volume and rebuild the containers
3. Ensure the `goapi` container does not exit due to an error
4. Check that the new tables and the data for the `user_roles` table is present in the database via Beekeeper

[Related Jira Ticket](https://communiteam.atlassian.net/browse/TEA-20)
